pages/dkp/konvoy/2.1/choose-infrastructure/aws/multi-account/index.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: layout.pug
+navigationTitle: Multiple AWS Accounts
+title: Multiple AWS Accounts
+menuWeight: 10
+excerpt: Leverage Multiple AWS Accounts for Kubernetes Cluster Deployments
+beta: false
+enterprise: false
+---
+
+# Leverage Multiple AWS Accounts for Kubernetes Cluster Deployments
+
+
+---
+
+## Objective
+
+You can leverage multiple AWS accounts in your organization to meet specific business purposes, reflect your organizational structure, or implement a multi-tenancy strategy. Specific scenarios include:
+
+* Implementing isolation between environment tiers such as development, testing, acceptance, and production.
+* Implementing separation of concerns between management clusters, and workload clusters.
+* Reducing the impact of security events and incidents.
+
+You can see the additional benefits of using multiple AWS accounts in the following [whitepaper](https://docs.aws.amazon.com/whitepapers/latest/organizing-your-aws-environment/benefits-of-using-multiple-aws-accounts.html)
+
+This document describes how to leverage the D2iQ Kubernetes Platform (DKP) to deploy a management cluster, and multiple workload clusters, leveraging multiple AWS accounts.
+
+## Assumptions
+
+This guide assumes you have some understanding of Cluster API concepts and basic DKP provisioning workflows on AWS.
+
+Cluster API Concepts - [cluster API concepts](https://cluster-api.sigs.k8s.io/user/concepts.html)
+
+Getting Started with DKP on AWS - [getting started on AWS](https://docs.d2iq.com/dkp/konvoy/2.1/choose-infrastructure/aws/)
+
+## Glossary
+
+* **Management cluster** - The cluster that runs in AWS and is used to create target clusters in different AWS accounts.
+* **Target account** - The account where the target cluster is created.
+* **Source account** - The AWS account where the CAPA controllers for the management cluster runs.
+
+## Prerequisites
+
+Before you begin deploying DKP on AWS, you must: 
+
+[prerequisites](https://docs.d2iq.com/dkp/konvoy/2.1/choose-infrastructure/aws/quick-start-aws/#configure-aws-prerequisites)
+
+## Deploy DKP on AWS
+
+1. Deploy a management cluster in your AWS source account.
+2. Configure a trusted relationship between source and target accounts.
+3. Create a management cluster:
+
+* **AWS:** [create Kubernetes AWS cluster](https://docs.d2iq.com/dkp/konvoy/2.1/choose-infrastructure/aws/quick-start-aws/#create-a-new-aws-kubernetes-cluster)
+* **EKS:** [create Kubernetes EKS cluster](https://docs.d2iq.com/dkp/konvoy/2.1/choose-infrastructure/eks/quick-start/#create-a-new-eks-kubernetes-cluster)
```

Comment:
Bullet 3. was supposed to be a sub-heading that walked the reader through how to deploy a management cluster. Also, we should remove the EKS cluster section since we don't support deploying a management cluster on EKS.

Reply:
done